libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -35,9 +35,10 @@ bool IRGeneratorForStatements::visit(VariableDeclarationStatement const& _varDec
 		if (decl)
 			m_context.addLocalVariable(*decl);
 
-	solUnimplementedAssert(_varDeclStatement.declarations().size() == 1, "");
 	if (Expression const* expression = _varDeclStatement.initialValue())
 	{
+		solUnimplementedAssert(_varDeclStatement.declarations().size() == 1, "");
```

Comment:
Why was this moved inside? It doesn't depend on `expression`

Reply:
Because the ``else`` case works just fine for multi-variable declarations - only the declaration-and-assignment case is unimplemented for the multi-variable case.